containers/neoantigen/Dockerfile

```diff
@@ -1,14 +1,14 @@
 FROM ubuntu:18.04
 
 LABEL maintainer="Allan Bolipata <bolipatc@mskcc.org>, Evan Biederstedt <evan.biederstedt@gmail.com>, Yixiao Gong <gongy@mskcc.org>" \
```

Comment:
@johnoooh already approved but you may want to add yourself (and also me, since I updated this file before).